src/actinia_core/core/common/endpoint_config.py

```diff
@@ -0,0 +1,87 @@
+# -*- coding: utf-8 -*-
```

Comment:
The two methods of this module are only used in all the endpoint classes, right? Then I would move this maybe to `src/actinia_core/rest/base/` ?